src/test/components/TrackThread.test.js

```diff
@@ -147,7 +162,15 @@ describe('timeline/TrackThread', function() {
   });
 
   it('can click a stack in the stack graph in normal call trees', function() {
-    const { getState, stackGraphCanvas, thread } = setup(getSamplesProfile());
+    const {
+      getState,
+      stackGraphCanvas,
+      thread,
+      ctx,
+      getFillRectCenterByIndex,
+    } = setup(getSamplesProfile());
+
+    const log = ctx.__flushDrawLog();
```

Comment:
ooh nice, TIL `__flushDrawLog`. I guess I've never used the mock canvas context before? :)

Reply:
Yeah, it's proved to be pretty helpful.